### ISSUE TITLE
"jupyterlab_bokeh@0.4.0" is not compatible with the current JupyterLab

### ISSUE THREAD

**Issue by jochym** [bokeh/jupyterlab_bokeh#33]
The extension is not compatible with JL 0.32 (beta2)

```
Conflicting Dependencies:
JupyterLab              Extension        Package
>=0.16.2 <0.17.0        >=0.15.0 <0.16.0 @jupyterlab/application
>=0.16.2 <0.17.0        >=0.15.0 <0.16.0 @jupyterlab/notebook
>=2.0.2 <3.0.0          >=1.0.1 <2.0.0   @jupyterlab/services
```

**Comment by jochym** [comment]
Still a problem for 0.4.1

**Comment by canavandl** [comment]
This was resolved in #37 and released in jupyterlab_bokeh=0.5.0. Please let open a new issue if you there's still a problem.
